tests/e2e_tests/test_unsubscribe_handler.py

```diff
@@ -9,18 +9,19 @@
 import pytest
 
 
+@pytest.mark.asyncio
 @pytest.mark.dependency(
     depends=[
         'tests/e2e_tests/test_subscribe_handler.py::test_subscribe_node_channel'],
     scope='session')
 @pytest.mark.order("last")
-def test_unsubscribe_node_channel(test_client):
+async def test_unsubscribe_node_channel(test_async_client):
```

Comment:
Does the client have to subscribe to the channel before unsubscribing or is this endpoint idempotent (essentially noop) if there was no subscription beforehand?

This could have been the reason why these tests were using synchronous client.

Reply:
See the dependency marker for the test.
The `unsubscribe` test will only execute after a successful execution of `test_subscribe_handler.py::test_subscribe_node_channel` test.

Reply:
You're right, I missed that - thanks!